app.rb

```diff
@@ -86,8 +85,6 @@ def list_rentals
     puts 'Enter the person ID:'
     id = gets.chomp
     puts 'Rentals:'
```

Comment:
- [ ] Same issue here as before. The `create_rentals` and `list_rentals` methods in the `app.rb` shouldn't ask a user for any input or validation but should simply either create the rental or list them. Kindly address this issue too 👍